apps/dfi/dfi.cpp

```diff
@@ -836,13 +836,15 @@ int main(int argc, const char *argv[])
 				case 1: return runDFI<PredMinFreq, PredMaxFreq, AminoAcid> (getArgumentValues(parser, 0), options.minFreq, options.maxFreq, options.maximal, options);
 				case 2: return runDFI<PredMinFreq, PredMaxFreq, Dna> (getArgumentValues(parser, 0), options.minFreq, options.maxFreq, options.maximal, options);
 			}
+            break;
```

Comment:
Although the code does not follow our guidelines, please indent to the same level as the line above